Chirp/ViewController.swift

```diff
@@ -7,13 +7,60 @@
 
 import UIKit
 
-class ViewController: UIViewController {
+class ViewController: UITabBarController {
 
+
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         // Do any additional setup after loading the view.
+  
+        
+        let vc1 = FirstViewVC()
+        let vc2 = SecondViewVC()
+        let vc3 = ThirdViewVC()
+        self.setViewControllers([vc1, vc2, vc3], animated: true)
+        
+        guard let items = self.tabBar.items else {
+            return
+        }
+        
+        let tabIconImages = ["ellipsis.message.fill", "person.2.fill", "squareshape.split.2x2"]
+        
+        for i in 0..<items.count {
+            items[i].image = UIImage(systemName: tabIconImages[i])
+        }
```

Comment:
Can we also set the tintColor and unSelected color to `.appBrown` and `.appBrownSecondary` You'll have access to both of these when you rebase with the latest development 
```swift
tabBar.tintColor = .brown
tabBar.unselectedItemTintColor = .lightGray
```